### PR TITLE
Adding flavors and loading right .env file

### PR DESCRIPTION
**Pull request by costa96**
PR for https://github.com/BasedHardware/Friend/issues/297.

Remember to run:
- `flutter clean`
- `flutter pub get`
- `flutter pub run build_runner build`

The app icon is the one automatically generated by flutter_flavorizr. Consider updating them with env-specific icons (For instance with a badge).

### CODE REVIEW

File: apps/AppWithWearable/flavorizr.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    ios:
+      bundleId: "com.friend.ios.dev"
+
+  production:
+    app:
+      name: "Friend"
+    android:
+      applicationId: "com.friend.ios"
+    ios:
+      bundleId: "com.friend.ios"
```
Comment (josancamon19) [inline apps/AppWithWearable/flavorizr.yaml:17]: The bundle ID for iOS is incorrect

**Review by josancamon19 — CHANGES_REQUESTED** [review]

File: apps/AppWithWearable/flavorizr.yaml (hunk 1 above)
Comment (josancamon19) [reply apps/AppWithWearable/flavorizr.yaml:17]: Also @costa96 could you upload screenshot of having both apps running locally? thanks

I'm not sure who worked on this @costa96 or @Hyla96

Comment (josancamon19) [reply apps/AppWithWearable/flavorizr.yaml:17]: For android and iOS

Comment (costa96) [reply apps/AppWithWearable/flavorizr.yaml:17]: Sorry, forgot to commit the changes with the right git account. It doesn't really matter, as you can guess I'm using two accounts. Regarding the bundle id, as far as I can see that's how the project is configured.
Screenshot 2024-06-19 at 07 28 54

**Comment by diegoasua** [comment]
We should make these three commands into a script
```
$ flutter clean
$ flutter pub get
$ flutter pub run build_runner build
```
Since they are run all the time

**Comment by josancamon19** [comment]
Closing this PR as this was assigned before on #306

**Pull request closed without merging**